src/components/Book.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button } from 'react-bootstrap';
+import './Book.css';
+
+const Book = ({
+  item_id, title, author, category, onRemove,
+}) => (
+  <div className="book-container p-4 border">
+    <div>
+      <ul className="list-unstyled">
+        <li>
+          <div>
+            <div className="book-data d-flex align-items-center justify-content-between ">
+              <div className="book-details">
+                <div className="d-flex flex-column">
+                  <span className="fw-bold gray">{category}</span>
+                  <span className="fw-bold title">{title}</span>
+                  <span className="text-primary">{author}</span>
+                </div>
+                <div className="book-buttons-container mt-3">
+                  <Button variant="light ps-0 text-primary">Comments</Button>
+                  <span className="mx-1">|</span>
+                  <Button variant="light text-primary" onClick={() => onRemove(item_id)}>
+                    Remove
+                  </Button>
+                  <span className="mx-1">|</span>
+                  <Button variant="light text-primary">Edit</Button>
+                </div>
+              </div>
```

Comment:
- [ ] As your previous code reviewer mentioned your design should match as much as possible and as per the requirement, your design should match **Exactly** with Zeplin design. Please remember, consider it a test of reproducing a given design. Kindly ensure the *add book button* is aligned on the right with the book cards like in the original design.

###  Yours
![Screenshot from 2023-09-03 01-57-53](https://github.com/axcell-0/bookstore/assets/66476329/84beccde-ac50-478a-b39e-2e956d33dd2a)

### Original
>   ![Screenshot_1](https://user-images.githubusercontent.com/65298048/265212916-5c3e5f6b-7bb3-458d-9913-4060da7010de.png)

Reply:
please sir my review are finished and I have try my best please dont let me fail because of a stylling please